tests/sunpath_test.py

```diff
@@ -0,0 +1,68 @@
+import math
+import pytest
+from pathlib import Path
+from ladybug.epw import EPW
+from ladybug.location import Location
+from ladybug.sunpath import Sunpath, Point3D, Vector3D
+from ladybug_vtk.to_vtk import create_polyline, create_points
+from ladybug_vtk.model_dataset import ModelDataSet
+from ladybug_vtk.model import Model
+
+
+def test_sunpath(temp_folder):
+
+    # Get location from epw file
+    epw = EPW('./tests/assets/weather/boston.epw')
+    location = epw.location
+
+    # Initiate sunpath
+    sp = Sunpath.from_location(location)
+
+    radius = 100
+    origin = Point3D(0, 0, 0)
+
+    # daily analemmas
+    polylines = sp.hourly_analemma_polyline3d(
+        origin=origin, daytime_only=True, radius=radius)
+    sp_polydata = [create_polyline(pl) for pl in polylines]
+    sp_dataset = ModelDataSet(name='Polylines', data=sp_polydata)
+
+    # monthly arcs
+    arcs = sp.monthly_day_arc3d()
+    arc_polylines = [arc.to_polyline(divisions=10)for arc in arcs]
+    arc_polydata = [create_polyline(arc) for arc in arc_polylines]
+    arc_dataset = ModelDataSet(name='Arcs', data=arc_polydata)
```

Comment:
For now, it is fine to use a Polyline but hopefully, we can support arcs at some point.

Reply:
https://github.com/ladybug-tools/ladybug-vtk/issues/3